_pages/mmyi.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: mmyi
+pageUrl: https://mmyi.github.io/
+timestamp: 2017-03-30
+---
```

Comment:
@mmyi add a line break, and remove the `https://` and `/`, that should allow the tests to pass!